ores/wsgi/util.py

```diff
@@ -35,23 +37,147 @@ def read_bar_split_param(request, param, default=None, type=str):
                          .format(param, str(e)))
 
 
+def format_error(error):
+    error_type = error.__class__.__name__
+    message = str(error)
+
+    return {'error': {'type': error_type, 'message': message}}
+
+
+def build_score_request(scoring_system, request, context_name=None, rev_id=None,
```

Comment:
Could move this to score_request.build_from_web_request

I wouldn't mind a docstring here, to remind me of the parameter types.